bubblewrap.c

```diff
@@ -499,9 +503,13 @@ privileged_op (int privileged_op_socket,
         die_with_error ("Can't mount proc on %s", arg1);
       break;
     case PRIV_SEP_OP_TMPFS_MOUNT:
-      if (mount ("tmpfs", arg1, "tmpfs", MS_MGC_VAL | MS_NOSUID | MS_NOEXEC, "mode=0755") != 0)
```

Comment:
devpts is internal to the sandbox anyway, and not visible from the outside, so its probably fine to do nothing about it.